source/sfp/2_soil/sfp_201_nl_oppervlaktemonster/07_werkwijze.Rmd

```diff
@@ -0,0 +1,185 @@
+Voor metingen of observaties: verwijs naar invulformulier (met versienummer; invulformulier toevoegen in bijlage; eventueel bepaalde zaken van invulformulier verduidelijken) en/of naar apparatuur en/of softwareprogramma indien gegevens digitaal worden ingevoerd (verwijs naar sip (standard instrument protocol, <protocol-code>-YYYY.NN) indien beschikbaar; indien geen sip beschikbaar, geef de nodige instructies om gegevens op gepaste wijzen in te kunnen geven).
+Voor staalnames: geef aan hoe de stalen bewaard, gelabeld en vervoerd moeten worden.
+Geef aan of de stalen voorbehandeld moeten worden.
+Voor digitale foto’s en/of andere digitale bestanden: geef aan hoe en waar deze bestanden moeten worden opgeslagen.
+Geef eventueel aan welke bestandsnamen aan de bestanden moeten gegeven worden.
+-->
+```
+### Invullen van het opnameformulier
+
+Per mengmonster wordt het opnameformulier volledig ingevuld en de aan- of afwezigheid van de relevante kenmerken genoteerd (voorbeeld: zie bijlage \@ref(bijlage-opnameformulier)).
+
+Het opnameformulier wordt, nadat het volledig werd ingevuld met onuitwisbare inkt, ingescand en digitaal bewaard op een centrale locatie (bvb. Google Drive).
+Het papieren formulier wordt overgemaakt aan de verantwoordelijke wetenschapper die alle opnameformulieren van de survey centraal bewaart.
+Elke staalnemer voert de informatie van het opnameformulier digitaal in zoals voorzien binnen het betreffende onderzoeksproject.
+
+### Nemen van foto's van het bemonsterd proefvlak
+
+Er worden twee foto's van het proefvlak genomen: één voor de bodembemonstering en één na de bemonstering (waar de bemonsteringspunten in het proefvlak maximaal zichtbaar zijn).
+Er wordt tenminste één foto van de boorkern in de guts genomen (detailfoto).
+Specifieke kenmerken (zie opnameformulier) worden ook gefotografeerd.
+
+Op de detailfoto wordt eveneens het ID van het proefvlak en datum van staalname weergegeven, samen met de schaalaanduiding d.m.v. bv.
+een vouwmeter.
+De digitale foto's worden zo snel mogelijk na de opname hernoemd met naam van het proefvlak en gefotografeerd kenmerk en op een centrale locatie bvb.
+Google Drive bewaard.
+De volgende principes worden gehanteerd bij de naamgeving van de foto's: de naam van de foto gaat van algemeen naar specifiek.
+Een voorbeeldje:
+
+-   De Omgevingsfoto: CODEPROEFVLAK_O
+-   De foto van het proefvlak/plot: CODEPROEFVLAK_PLOT
+-   De foto van het 0 -- 10 cm staal: CODEPROEFVLAK_M01
+
+### Identificatie (etikettering) van de monsters
+
+De volgende informatie moet minimaal op de recipiënten aanwezig zijn.
+Bij voorkeur wordt deze informatie vooraf (indien reeds beschikbaar) op 'heavy duty' etiketten afgedrukt.
+
+-   Plaatsnaam (hoofdgemeente en toponiem)
+-   Unieke veldidentificatiecode van het proefvlak
+-   Bemonsteringsdiepte (in dit geval 0-10 cm)
```

Comment:
De vaste 0-10 cm waarden verschijnen nog hier en daar in het protocol=> nog vervangen door de parameters?

**In 07_werkwijze:**

Identificatie (etikettering) van de monsters

> Bemonsteringsdiepte (in dit geval 0-10 cm)

vervangen door "Bemonsteringsdiepte (bvb. 0-10 cm)" of "Bemonsteringsdiepte (in dit geval params$begin_diepte_cm` -- `r params$eind_diepte_cm` cm) "

**In 10_samenvatting:**

> Neem van elk monster het gewenste materiaal (bvb. over een diepte van de eerste 10 cm) en breng in een mengemmertje 

"(bvb. over een diepte van de eerste 10 cm)" zouden we kunnen vervangen door zoiets "(in dit geval van params$begin_diepte_cm` -- `r params$eind_diepte_cm` cm diep)"